src/main/java/com/vg/hm/samples/usermanagementsvc/config/JaxRSConfig.java

```diff
@@ -13,6 +14,7 @@ public JaxRSConfig() {
         log.info("Initializing Rest Services...");
         packages("com.vg.hm.samples.usermanagementsvc");
         register(UserV1RestService.class);
```

Comment:
If you use the packages("com.vg.hm.samples.usermanagementsvc") it means it will scan any class that uses JaxRS annotations under this package. As long as your RestServices are within this package, you shouldn't need to register them one by one. This means, you can either keep the line 15, and remove the line 16 and 17, or remove the line 15 and keep the other ones. My recommendation is remove the packages one so it doesn't need to scan the whole thing.

Reply:
removed the packages and will register each REST service.